code/obj/critter/misc.dm

```diff
@@ -487,22 +489,43 @@
 			else
 				continue
 
+	attackby(obj/item/W as obj, mob/M as mob)
+		if(istype(W, /obj/item/reagent_containers/food/snacks) && !(M in src.friends))
+			if(prob(20))
+				src.visible_message("<span class='notice'>[src] chitters happily at the [W], it will no longer attack [M]!</span>")
+				friends += M
+				playsound(src.loc, "sound/misc/bugchitter.ogg", 50, 0)
+				src.task = "thinking"
+			else
+				src.visible_message("<span class='notice'>[src] hated the [W]! It bit [M]'s hand!</span>")
+				take_bleeding_damage(M, M, rand(6,12), DAMAGE_STAB, 1)
```

Comment:
This makes the mob deal damage to itself instead of the scorpion being the damage dealer. Intended?